sdk/core/Azure.Core/tests/ClientDiagnosticsTests.Net50.cs

```diff
@@ -329,6 +338,10 @@ public void CanSetActivitySourceAndDiagnosticSourceActivitiesTogether()
             CollectionAssert.Contains(diagnosticSourceActivity.Tags, new KeyValuePair<string, string>("Attribute2", "2"));
             CollectionAssert.Contains(diagnosticSourceActivity.Tags, new KeyValuePair<string, string>("Attribute3", "3"));
 
+            // Since both ActivitySource and DiagnosticSource listeners are used, we should see the az.schema_url tag set even in diagnostic source because they use the same
```

Comment:
I think this is expected.. @lmolkova please keep me honest

Reply:
yep, if both are used, there is nothing we can do to keep DiagSource path unchanged. But it's not a common (or valid) scenario to have both.